site/sfguides/src/servicenow_to_snowflake_connector/servicenow_to_snowflake_connector.md

```diff
@@ -0,0 +1,364 @@
+
+- A view named table_name__view_with_deleted that contains the same data as table_name__view as well as rows for records that have been deleted in ServiceNow.
+
+- A table table_name__event_log that contains the history of changes made to records in ServiceNow.
+
+ To query from the raw data, check out [Accessing the raw data](https://other-docs.snowflake.com/en/connectors/servicenow/servicenow-accessing-data.html#accessing-the-raw-data). To query the views (recommended), check out [Accessing the flattened data](https://other-docs.snowflake.com/en/connectors/servicenow/servicenow-accessing-data.html#accessing-the-flattened-data).
+
+### Use this query to identify number of incidents raised by month and priority
+Here's a little test query for you to identify the number of incidents raised by month and priority. Other example queries are provided on the Snowflake Connector for ServiceNow page in the Marketplace.
+
+```SQL
+USE ROLE ACCOUNTADMIN;
+USE DATABASE SERVICENOW_DEST_DB;
+USE SCHEMA DEST_SCHEMA;
+
+WITH T1 AS (
+    SELECT
+    DISTINCT
+        T.NUMBER AS TICKET_NUMBER
+        ,T.SHORT_DESCRIPTION
+        ,T.DESCRIPTION
+        ,T.PRIORITY
+        ,T.SYS_CREATED_ON AS CREATED_ON
+        ,T.SYS_UPDATED_ON AS UPDATED_ON
+        ,T.CLOSED_AT
+    FROM
+      TASK__VIEW T
+     LEFT JOIN 
+          INCIDENT__VIEW I 
+          ON I.SYS_ID = T.SYS_ID -- ADDITIONAL INCIDENT DETAIL
+      LEFT JOIN 
+          SYS_AUDIT_DELETE__VIEW DEL 
+          ON T.SYS_ID = DEL.DOCUMENTKEY -- THIS JOIN HELPS IDENTIFY DELETED TICKETS  
+    WHERE
+        DEL.DOCUMENTKEY IS NULL --  THIS CONDITION HELPS KEEP ALL DELETED RECORDS OUT
+    AND
+        I.SYS_ID IS NOT NULL -- THIS CONDITION HELPS KEEP JUST THE INCIDENT TICKETS
+)
+SELECT
+    YEAR(CREATED_ON) AS YEAR_CREATED
+    ,MONTH(CREATED_ON) AS MONTH_CREATED
+    ,CONFIGURATION_ITEM AS APPLICATION
+    ,PRIORITY
+    ,COUNT(DISTINCT TICKET_NUMBER) AS NUM_INCIDENTS
+FROM
+    T1
+GROUP BY
+    YEAR_CREATED
+    ,MONTH_CREATED
+    ,PRIORITY
+ORDER BY
+    YEAR_CREATED
+    ,MONTH_CREATED
+    ,PRIORITY
+;
+```
+## Setting the monitoring role permissions
+If you would like to monitor errors, run stats, connector stats, enabled tables, you can set up a ServiceNow monitoring role that allows access to the views in the connector database.  For example, run the following in a worksheet (and then use the role):
+```SQL
+USE ROLE accountadmin;
+CREATE ROLE IF NOT EXISTS servicenow_monitor_role ;
+GRANT IMPORTED PRIVILEGES ON DATABASE SNOWFLAKE_CONNECTOR_FOR_SERVICENOW TO ROLE servicenow_monitor_role;
+GRANT USAGE ON SCHEMA PUBLIC TO ROLE servicenow_monitor_role; 
+GRANT SELECT ON FUTURE TABLES IN SCHEMA PUBLIC TO ROLE servicenow_monitor_role;
+GRANT SELECT ON FUTURE VIEWS IN SCHEMA PUBLIC TO ROLE servicenow_monitor_role;
+GRANT SELECT ON ALL TABLES IN SCHEMA PUBLIC TO ROLE servicenow_monitor_role;
+GRANT SELECT ON ALL VIEWS IN SCHEMA PUBLIC TO ROLE servicenow_monitor_role;
```

Comment:
As far as I know, it is not possible to perform such grants on a mounted database (a database created from a share). `GRANT IMPORTED PRIVILEGES ON DATABASE SNOWFLAKE_CONNECTOR_FOR_SERVICENOW TO ROLE servicenow_monitor_role;` and grant on warehouse should be sufficient.

Also granting this role to `accountadmin` is rather unnecessary as accountadmin is already the owner of the SNOWFLAKE_CONNECTOR_FOR_SERVICENOW database